docs/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception.md

```diff
@@ -6,52 +6,41 @@ helpviewer_keywords:
 ms.assetid: db4630b3-5240-471a-b3a7-c7ff6ab31e8d
 ---
 # How to: Catch a non-CLS Exception
-Some .NET languages, including C++/CLI, allow objects to throw exceptions that do not derive from <xref:System.Exception>. Such exceptions are called *non-CLS exceptions* or *non-Exceptions*. In Visual C# you cannot throw non-CLS exceptions, but you can catch them in two ways:  
+Some .NET languages, including C++/CLI, allow objects to throw exceptions that do not derive from <xref:System.Exception>. Such exceptions are called *non-CLS exceptions* or *non-Exceptions*. In C# you cannot throw non-CLS exceptions, but you can catch them in two ways:  
   
--   Within a `catch (Exception e)` block as a <xref:System.Runtime.CompilerServices.RuntimeWrappedException>.  
+-   Within a `catch (RuntimeWrappedException e)` block.
   
      By default, a Visual C# assembly catches non-CLS exceptions as wrapped exceptions. Use this method if you need access to the original exception, which can be accessed through the <xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A> property. The procedure later in this topic explains how to catch exceptions in this manner.  
```

Comment:
unrelated to your PR: add ?displayProperty=nameWithType to \<xref:System.Runtime.CompilerServices.RuntimeWrappedException.WrappedException%2A>. I think that the other two occurrences of WrappedException might be clearer with ?displayProperty=nameWithType as well.

Reply:
@rpetrusha Done.